<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>631</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1130</width>
    <height>631</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1130</width>
    <height>631</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>39</y>
      <width>821</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 143, 143);
border-color: rgb(13, 13, 13);</string>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#f9f9f9;&quot;&gt; 文件夹&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>842</x>
      <y>36</y>
      <width>256</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>851</x>
      <y>189</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加文件夹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>981</x>
      <y>189</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除文件夹</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>861</x>
      <y>229</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>扫描子文件夹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>380</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>停止扫描</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../../../../21161/Downloads/b6e8520c5c986ce240d59824a0e9c89b.jpeg</normaloff>../../../../../../../../../21161/Downloads/b6e8520c5c986ce240d59824a0e9c89b.jpeg</iconset>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>951</x>
      <y>304</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Average Hash</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Color Distribution</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Perceptual Hash</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Different Hash</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>310</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;方  法：&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>380</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string> 开始扫描</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../../../../21161/Downloads/bf9e346b5483aba4aebe5b8b5f13d53a.jpeg</normaloff>../../../../../../../../../21161/Downloads/bf9e346b5483aba4aebe5b8b5f13d53a.jpeg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>340</y>
      <width>95</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Similarity：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>953</x>
      <y>271</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>871</x>
      <y>271</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>原图路径：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>272</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>340</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>390</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;预览&lt;/p&gt;&lt;p&gt;原图：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>72</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;预览&lt;/p&gt;&lt;p&gt;相似图：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1130</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Y">
    <property name="title">
     <string>文件(&amp;Y)</string>
    </property>
    <addaction name="actionNew_Projrct"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave"/>
    <addaction name="actionAdd"/>
    <addaction name="actionFilter"/>
    <addaction name="actionSearch"/>
   </widget>
   <widget class="QMenu" name="menu_Z">
    <property name="title">
     <string>帮助(&amp;Z)</string>
    </property>
   </widget>
   <addaction name="menu_Y"/>
   <addaction name="menu_Z"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Projrct">
   <property name="text">
    <string>New </string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open </string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="text">
    <string>Filter</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionStart_Scan">
   <property name="text">
    <string>Start Scan</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
